### DISCUSSION TITLE
Performance tests over the ZFS versions

### DISCUSSION THREAD

**Discussion by interduo** [openzfs/zfs#15337] [Ideas]
Many people asks me if it's worth to upgrade big NASes with ZFS concerning the performance.

I tried to search that. There is not much information in the Internet.
Maybe we should do some performance test suite and put results in some webarea. The test should be on the same hardware - just using different ZFS version+latest stable kernel.

What You think about it?

**Comment by gdevenyi** [comment]
A standardized set of fio test configurations is the way to go.

https://fio.readthedocs.io/en/latest/fio_doc.html

I've never seen anyone comprehensively them against ZFS
